buildkite/test-e2e.dockerfile

```diff
@@ -3,5 +3,5 @@ FROM mrcide/mint-shared-build-env:$GIT_ID
 
 RUN cd app/static && npx playwright install chromium
 
-CMD ./gradlew app:bootRun & sleep 60 && \
+CMD ./gradlew app:bootRun & sleep 90 && \
```

Comment:
This now matches `test-integration.dockerfile` and is necessary to prevent timeouts since baseline options were extended and time taken to initialise database increased accordingly